Source/WebKit/UIProcess/API/glib/WebKitSettings.h.in

```diff
@@ -530,6 +530,13 @@ WEBKIT_API void
 webkit_settings_set_enable_webrtc                              (WebKitSettings *settings,
                                                                 gboolean enabled);
 
+WEBKIT_API gboolean
+webkit_settings_get_enable_media_session                       (WebKitSettings *settings);
+
+WEBKIT_API void
+webkit_settings_set_enable_media_session                       (WebKitSettings *settings,
+                                                                gboolean enabled);
```

Comment:
Alignment is wrong here

Reply:
Are you talking about the second argument? I used same alignment as above.

Reply:
Should be:

```suggestion
                                                                gboolean        enabled);
```